src/services/findTransferSteps.js

```diff
@@ -1,84 +1,10 @@
-import HubContract from 'circles-contracts/build/contracts/Hub.json';
-import TokenContract from 'circles-contracts/build/contracts/Token.json';
 import findTransferSteps from '@circles/transfer';
 import { performance } from 'perf_hooks';
 
-import web3 from './web3';
-import EdgeUpdateManager from './edgesUpdate';
-import tasks from '../tasks';
-import submitJob from '../tasks/submitJob';
-import { minNumberString } from '../helpers/compare';
-import loop from '../helpers/loop';
 import { EDGES_FILE_PATH, PATHFINDER_FILE_PATH } from '../constants';
 
 const DEFAULT_PROCESS_TIMEOUT = 1000 * 60;
 
-const hubContract = new web3.eth.Contract(
-  HubContract.abi,
-  process.env.HUB_ADDRESS,
-);
-
-async function isStepValid(
-  tokenAddress,
-  tokenOwner,
-  sender,
-  receiver,
-  plannedLimit,
-) {
-  // Get send limit
-  const limit = await hubContract.methods
-    .checkSendLimit(tokenOwner, sender, receiver)
-    .call();
-  // Get Token balance
-  const tokenContract = new web3.eth.Contract(TokenContract.abi, tokenAddress);
```

Comment:
are these functions not being use anywhere else?

Reply:
Do you both functions together? What do you mean?

Reply:
actually I just realised these were moved to a different file, sorry!